source/ui/Components/Memory/MemoryContent.qml

```diff
@@ -0,0 +1,28 @@
+/* C++ Assembler Interpreter
+* Copyright (C) 2016 Chair of Computer Architecture
+* at Technical University of Munich
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/>.*/
+
+import QtQuick 2.6
+import QtQuick.Controls 1.5
+
+import Theme 1.0
+
+TableViewColumn {
+  role: "BinaryData" + numberOfBits
+  movable: false
+  resizable: true
+  width: Theme.memory.address.width
```

Comment:
I don't know why, but I can't resize the columns...

It would be nice to set the width to the maximum available width without having to scroll, as now there is a lot of unused space if there is only one column.